packages/AutoTDD-Core.package/ATDDNullDataSource.class/instance/repository.st

```diff
@@ -1,4 +1,4 @@
 accessing
 repository
 	
-	^ String empty
\ No newline at end of file
+	^ ''
```

Comment:
What was wrong with `String empty`?

Reply:
Nothing, but we should only use one of them, we can decide for one and change all occurrences sometime